spec/views/publishers/vacancies/job_applications/index.html.slim_spec.rb

```diff
@@ -1,12 +1,12 @@
 require "rails_helper"
 
 RSpec.describe "publishers/vacancies/job_applications/index" do
-  let(:organisation) { build_stubbed(:school) }
-  let(:publisher) { build_stubbed(:publisher, organisations: [organisation]) }
+  let(:organisation) { create(:school) }
+  let(:publisher) { create(:publisher, organisations: [organisation]) }
```

Comment:
There should be no need to convert these to create's - build_stubbed should always work for view specs, as they have id, created_at and relationships

Reply:
the job_applications are retrieved with an active relation hence the change